### PR TITLE
Fix timeline resize

### PR DESCRIPTION
**Pull request by lchan131**
Fix: When screen shrinks, timeline expands indefinitely

### CODE REVIEW

**Review by MichaelKovalchuk — APPROVED** [review]

File: capture/d2l-capture-producer/src/d2l-capture-producer-timeline.js [hunk 1]
```diff
@@ -1054,8 +1054,10 @@ class CaptureProducerTimeline extends RtlMixin(InternalLocalizeMixin(LitElement)
 
 	_updateRelativeWidth() {
 		const rootContainer = this.shadowRoot.querySelector('.d2l-video-producer-timeline');
-		const {value: rootWidth} = this._parseWidth(getComputedStyle(rootContainer).getPropertyValue('width'));
-		this._widthPixels = rootWidth;
+		const {value: rootOuterWidth} = this._parseWidth(getComputedStyle(rootContainer).getPropertyValue('width'));
+		const rootBorderOffset = 2;
+		const rootInnerWidth = rootOuterWidth - rootBorderOffset;
```
Comment (MarquisLP) [inline capture/d2l-capture-producer/src/d2l-capture-producer-timeline.js:1059]: So subtracting 2 from the width prevents the timeline from expanding indefinitely in smaller window sizes? Can we add a comment explaining why this works?

Comment (lchan131) [reply capture/d2l-capture-producer/src/d2l-capture-producer-timeline.js:1059]: Sure.

File: capture/d2l-capture-producer/src/d2l-capture-producer-timeline.js [hunk 2]
```diff
@@ -1054,8 +1054,13 @@ class CaptureProducerTimeline extends RtlMixin(InternalLocalizeMixin(LitElement)
 
 	_updateRelativeWidth() {
 		const rootContainer = this.shadowRoot.querySelector('.d2l-video-producer-timeline');
-		const {value: rootWidth} = this._parseWidth(getComputedStyle(rootContainer).getPropertyValue('width'));
-		this._widthPixels = rootWidth;
+		const {value: rootOuterWidth} = this._parseWidth(getComputedStyle(rootContainer).getPropertyValue('width'));
+		const rootBorderOffset = 2;
```
Comment (MarquisLP) [inline capture/d2l-capture-producer/src/d2l-capture-producer-timeline.js:1058]: Nit: for consistency, we should move this hardcoded value to src/util/constants.js.

**Review by MarquisLP — APPROVED** [review]
Thanks for explaining the infinite loop on the observer!

**Review by MichaelKovalchuk — APPROVED** [review]

**Review by MarquisLP — APPROVED** [review]

**Pull request merged**